network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_authors.html

```diff
@@ -5,7 +5,7 @@
 <div class="blog-authors d-flex flex-wrap {% if show_full_info %}flex-column mb-4{% else %}align-items-center{% endif %}">
   <div class="author-images d-flex flex-wrap">
   {% for author in authors %}
-    <div class="author-image d-flex align-items-center justify-content-center">
+    <div class="author-image d-flex align-items-center justify-content-center {% if author.image %}no-background{% endif %}">
```

Comment:
As people will have pictures most of the time, let's reverse the logic so that we only add a class for authors without an image, which also simplifies the CSS by not needing a `:not`

```suggestion
    <div class="author-image d-flex align-items-center justify-content-center {% if author.image is None %}no-avater{% endif %}">
```